Никишин В.Л. Лаб. 1 Группа 6312

h0shi-tech opened the pull request:
**ФИО:** Никишин Владислав 
**Номер группы:** 6312  
**Номер лабораторной:** 1  
**Краткое описание исходного кода:** Я взял(а) исходный код из *своего старого задания по программированию*. 
**Краткое описание проделанной работы:** *Привел грязный код к PEP-8*.

IlyaOv commented on lab_4/main.py:
```diff
@@ -0,0 +1,668 @@
+    if command == 1:
+        new_name = input("New Full Name: ")
+        users[account_number]["full_name"] = new_name
+        logging.info(f"update_information: Updated full name to '{new_name}' for account number {account_number}.")
```
Почему решили поменять способ форматирования?
Предлагаю остановиться на %.
https://docs.python.org/3/howto/logging.html#optimization
https://docs.python.org/3/howto/logging-cookbook.html#formatting-styles

IlyaOv commented:
Зачтено

h0shi-tech committed:
Грязный код

ФИО: Никишин Владислав Леонидович
Номер группы: 6312
Номер лабораторной: 1
Краткое описание исходного кода: Взял случайный код из интернета

h0shi-tech committed:
Код приведенный к PEP-8

Что было исправлено:
- привёл файл к стандарту PEP-8 (отступы, пробелы, длина строк ≤ 79 символов);
- переименовал функции, методы и переменные в snake_case;
- переименовал классы в CamelCase, согласно PEP-8;
- добавил docstring-и для всех классов и функций;
- перенёс длинные выражения на несколько строк;
- аккуратно оформил многострочные print;
- структурировал импорт: один импорт — одна строка;
- заменил ручное open().close() на контекстный менеджер with;
- убрал лишние пустые строки и привёл структуру файла в порядок;
- добавил f-строки вместо ручной конкатенации в print;
- отформатировал огромные списки аргументов;
- сделал main() чистым и читаемым.
